pkg/lib/oauth/handler/resolve.go

```diff
@@ -75,6 +75,14 @@ func validateRedirectURI(client *config.OAuthClientConfig, httpOrigin httputil.H
 		}
 	}
 
+	// Implicitly allow URIs for all whitelisted domains in https
+	for _, domain := range domainWhitelist {
+		origin := fmt.Sprintf("https://%s", domain)
```

Comment:
Use the injectable`httputil.HTTPProto` here, instead of hard coding `https`.

Reply:
Updated, thanks